man/systemd-tmpfiles.xml

```diff
@@ -227,6 +230,24 @@
     <programlisting>systemd-tmpfiles --remove --create</programlisting>
   </refsect1>
 
+  <refsect1>
+    <title>Environment</title>
+
+    <variablelist class='environment-variables'>
+      <xi:include href="common-variables.xml" xpointer="log-level" />
+      <xi:include href="common-variables.xml" xpointer="log-color" />
+      <xi:include href="common-variables.xml" xpointer="log-time" />
+      <xi:include href="common-variables.xml" xpointer="log-location" />
+      <xi:include href="common-variables.xml" xpointer="log-target" />
```

Comment:
doesn't xinclude/xpointer have some way to say: `… xpointer=log-*"…`? that'd be so much ncier

Reply:
It would. But I don't think there's any support for that.